tests/run_not_found.out

```diff
@@ -0,0 +1,2 @@
+NotFound: Cannot resolve module "./not-a-valid-remote-module.ts" from "."
+    at <unknown>:1:1
```

Comment:
I guess is it really unknown?  This could still be more clearer.  I think if we simply passed an empty stack frame vector.  It feels like the stack frame is more confusing than not, since it isn't anything in the runtime that is interpreting it, therefore there is no code reference.

Reply:
Yeah it might be more confusing... The problem is there is not really a lot of useful information from `DenoError`... Probably I should just pass in an empty frame vector instead...